app/assets/stylesheets/styles.scss

```diff
@@ -37,10 +130,10 @@ html {
                 font-size: 2em;
                 align-items: center;
                 a {
-                    font-family: $heading-font;
+                    font-family: $star-wars;
                     font-variant: small-caps;
                     text-decoration: none;
-                    color: black;
+                    color: white;
                 }
             }
             .hamburger {
```

Comment:
remove the hamburger class entirely when you're at it...just for cleanup purposes, not related to your change.